patterns/2-structured/innersource-license.md

```diff
@@ -66,6 +67,8 @@ The mentioned collaboration challenges include:
 
 It is worth mentioning that so far the software shared under this InnerSource license is mostly tooling, infrastructure, and tools lower in the stack.
 
+- **Robert Bosch GmbH** 
```

Comment:
```suggestion
- ** Deutsche Bahn AG** 
- **Robert Bosch GmbH** 
```

Reply:
I suggest moving the list to the top of the section before the case details.

Reply:
Good call @fioddor!

Formatting on this PR is not final yet, which is why we kept it as a "Draft".

Actually this PR happened as part of a (super rare) in-person conversation between @gruetter and me, so in a sense this is more of a mental note :)

Reply:
I changed the formatting now.